twofactor_privacyidea/templates/challenge.php

```diff
@@ -3,6 +3,8 @@
     $u2fSignRequest = $_["u2fSignRequest"];
     script('twofactor_privacyidea', 'u2f-api');
     script('twofactor_privacyidea', 'u2f');
+} else {
+    $u2fSignRequest = false;
```

Comment:
Don't we need this variable in the later course?

Reply:
There is a ``if ($u2fSignRequest)`` later on. If $u2fSignRequest is uninitialized, everything works fine, but the warning from #12 is emitted.

Reply:
Hm, and if it is set to ``false`` it does not work fine?
I do not understand why to remove this.